Makefile

```diff
@@ -3,12 +3,62 @@
 # Moreover, any project using sandbox as dependency won't build unless
 # there's Foundry installed on the machine.
 
-.PHONY: artifacts
+.PHONY: artifacts cairo-lang local-contracts starkgate-contracts-latest starkgate-contracts-old
+
+CAIRO_LANG_COMMIT_HASH="8e11b8cc65ae1d0959328b1b4a40b92df8b58595"
+STARKGATE_CONTRACTS_COMMIT_HASH="45941888479663ac93e898cd7f8504fa9066c54c"
```

Comment:
should we use a release instead?

Reply:
commit looks good in my opinion.

Reply:
why though? isn't a release cleaner? do you imagine a scenario where we would need to be on specific commits and not releases?

Reply:
Right now we are using commit hash as you can see in the makefile.